src/lib/svg-views.tsx

```diff
@@ -31,19 +31,19 @@ import { injectable } from 'inversify';
 
 @injectable()
 export class SvgViewportView implements IView {
-    render(model: Readonly<ViewportRootElement>, context: RenderingContext): VNode {
+    render(model: Readonly<ViewportRootElement>, context: RenderingContext, args?: IViewArgs): VNode {
         const transform = `scale(${model.zoom}) translate(${-model.scroll.x},${-model.scroll.y})`;
         return <svg>
             <g transform={transform}>
-                {context.renderChildren(model)}
+                {context.renderChildren(model, args)}
```

Comment:
Passing the `args` explicitly is no longer necessary, right?
See also other views below.

Reply:
Right, they are not needed anymore. I must have missed those, as I believe I removed them anywhere else. In the new commit I removed them from the `svg-views.tsx` too. Thanks!